### PR TITLE
Add new config file to pick-up new odyssey golden image V2

### PR DESCRIPTION
**Pull request by sumitk56**
Change-Id: Idd7b7ba7c8bebbab7bf04839bc43d8ced38a17d8

### CODE REVIEW

File: imageBuild/configs/odyssey/dd1/ody_pnor_dd1_image_configv2 [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    # Name of the sbe Tools tar
+    'sbeTools'  : 'sbe_tools.tar.gz',
+
+    # 'binaries' contains the released binary images (optional), expected to be a git repository
+    'binaries'    :  {
+        'repository' : ['git clone https://github.com/open-power/hostboot-binaries.git --branch master-p10',],
+
+        #          (File to extract into delivered binaries, branch/commit/tag  to use (empty string to use latest commit)),
+        'files' : [('sbe_images/odyssey_dd1_0/golden/v2/golden_odyssey_nor_DD1.img.tar.gz',''),
+                   ('sbe_images/odyssey_dd1_0/golden/v2/ody_sbe_golden_debug.tar.gz',''),
```
Comment (akhis023) [inline imageBuild/configs/odyssey/dd1/ody_pnor_dd1_image_configv2]: We wanted to remove debug.tar as it's not required.

File: imageBuild/configs/odyssey/dd1/ody_pnor_dd1_image_configv2 [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+    # Name of the sbe Tools tar
+    'sbeTools'  : 'sbe_tools.tar.gz',
+
+    # 'binaries' contains the released binary images (optional), expected to be a git repository
+    'binaries'    :  {
+        'repository' : ['git clone https://github.com/open-power/hostboot-binaries.git --branch master-p10',],
+
+        #          (File to extract into delivered binaries, branch/commit/tag  to use (empty string to use latest commit)),
+        'files' : [('sbe_images/odyssey_dd1_0/golden/v2/golden_odyssey_nor_DD1.img.tar.gz',''),
```
Comment (akhis023) [inline imageBuild/configs/odyssey/dd1/ody_pnor_dd1_image_configv2]: I am not reviewing the complete file as its a copy and the only change in here is the golden image paths.

**Review by akhis023 — CHANGES_REQUESTED** [review]
1. There is a separate config for odysseylab image. Don't we want a v2 version of the golden for odylab? 
2. File naming could have been ody_pnor_dd1_image_config_v2

**Comment by sumitk56** [comment]
> 1. There is a separate config for odysseylab image. Don't we want a v2 version of the golden for odylab?
> 2. File naming could have been ody_pnor_dd1_image_config_v2

1. Yes will do for ody lab as well
2. As mentioned in other place, I did it intentionally so we don't rename/touch our existing config filename when not using v2 and it continue to work as-is

File: imageBuild/configs/odyssey/dd1/ody_pnor_dd1_image_configv2 (hunk 1 above)
Comment (rajez133) [reply imageBuild/configs/odyssey/dd1/ody_pnor_dd1_image_configv2]: +1

**Review by rajez133 — CHANGES_REQUESTED** [review]

File: imageBuild/configs/odyssey/dd1/ody_pnor_dd1_image_configv2 (hunk 1 above)
Comment (sumitk56) [reply imageBuild/configs/odyssey/dd1/ody_pnor_dd1_image_configv2]: > We wanted to remove debug.tar as it's not required.

yes will do

**Review by akhis023 — APPROVED** [review]

**Pull request closed without merging**